lib/src/differs/list_differ.dart

```diff
@@ -206,7 +206,7 @@ List<ListChangeRecord/*<E>*/ > _calcSplices/*<E>*/(
   oldEnd -= suffixCount;
 
   if (currentEnd - currentStart == 0 && oldEnd - oldStart == 0) {
```

Comment:
fix the _calcSplices to remove the generic method comments and use generic method syntax?